packages/core/src/amazonqFeatureDev/controllers/chat/controller.ts

```diff
@@ -306,6 +307,14 @@ export class FeatureDevController {
                         },
                     ],
                 })
+            } else if (err instanceof ZipFileError) {
+                this.messenger.sendErrorMessage(
+                    '',
+                    message.tabID,
+                    0,
```

Comment:
Is it good to not provide retry for this, as this conversation will be counted ?

Reply:
As many retries the user will do will not help, since this only can happen when our toolkit has a bug, then we need to have another version of the toolkit